examples/safety_and_security/retail_agent/README.md

```
@@ -0,0 +1,789 @@
+
+From the example directory, run a single query:
+
+```bash
+cd ./examples/risk_and_security/retail_agent
+```
+
+```bash
+nat run --config_file configs/config.yml --input "Email From: john@email.com\nContent: What garden trowels do you have?"
+```
+
+**Expected Workflow Output**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)